manifests/cf-manifest/scripts/generate-cf-secrets.rb

```diff
@@ -14,6 +14,7 @@
   "bulk_api_password" => :simple,
   "nats_password" => :simple,
   "router_password" => :simple,
+  "route_services_secret" => :simple,
```

Comment:
The docs mention that this needs to be a "strong passphrase". We should investigate what that really means, and if necessary introduce a stronger scheme for the password generator.